### PR TITLE
Add unit testing Proof-of-Concept

### PR DESCRIPTION
**Pull request by HansenSven**

### CODE REVIEW

File: Epsilon.Tests/Epsilon.Tests.csproj [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.3.2" />
+    <PackageReference Include="Moq" Version="4.18.4" />
+    <PackageReference Include="xunit" Version="2.4.2" />
```
Comment (Typiqally) [inline Epsilon.Tests/Epsilon.Tests.csproj:14]: Is there any reason to use xunit over NUnit over here?

File: Epsilon.Canvas.Tests/Services/AssignmentHttpServiceTests.cs [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+        public AssignmentHttpServiceTests()
+        {
+            _httpClient = new HttpClient();
+            _paginatorHttpServiceMock = new Mock<IPaginatorHttpService>();
+            _assignmentHttpService = new AssignmentHttpService(_httpClient, _paginatorHttpServiceMock.Object);
+        }
+
+        [Fact]
+        public async Task GetAll_ReturnsAssignments()
```
Comment (Typiqally) [inline Epsilon.Canvas.Tests/Services/AssignmentHttpServiceTests.cs]: Can we use acceptance criteria notation here? Given... when... then

File: Epsilon.Tests/ExportDataPackagerTests.cs [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Epsilon.Tests
+{
+    public class ExportDataPackagerTests
+    {
+        [Fact]
+        public async Task GetExportData_ReturnsExportData()
```
Comment (Typiqally) [inline Epsilon.Tests/ExportDataPackagerTests.cs]: Given... when... then

File: Epsilon.Tests/Exporters/ExcelModuleExporterTests.cs [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+using Epsilon.Export.Exporters;
+using Microsoft.Extensions.Options;
+using System.Reflection;
+
+namespace Epsilon.Tests.Exporters
+{
+    public class ExcelModuleExporterTests
+    {
+        [Fact]
+        public async Task Export_WithValidData_ReturnsStreamWithExpectedContent()
```
Comment (Typiqally) [inline Epsilon.Tests/Exporters/ExcelModuleExporterTests.cs]: Given... when... then

**Review by Typiqally — CHANGES_REQUESTED** [review]

**Comment by NealGeilen** [comment]
image
Tests run in Rider

**Review by NealGeilen — APPROVED** [review]

**Review by Typiqally — APPROVED** [review]

**Pull request merged**